src/dfx/assets/new_project_node_files/webpack.config.js

```diff
@@ -37,7 +37,7 @@ const canisterEnvVariables = initCanisterEnv();
 
 const isDevelopment = process.env.NODE_ENV !== "production";
 
-const frontendDirectory = "{project_name}_assets";
+const frontendDirectory = "{project_name}_frontend";
 
 const asset_entry = path.join("src", frontendDirectory, "src", "index.html");
```

Comment:
frontend_entrypoint (this is the index.js/jsx/ts/tsx file)

(Replace reference in rest of file)

Reply:
^^ still needs renaming